frontend/src/metabase/admin/permissions/components/PermissionsTable/PermissionsTable.styled.tsx

```diff
@@ -80,7 +82,7 @@ export const EntityNameLink = styled(Link)`
   display: inline;
   font-weight: 700;
   text-decoration: underline;
-  color: ${color("admin-navbar")};
+  color: ${() => color("admin-navbar")};
```

Comment:
@WiNloSt I think it's because of this logic I mentioned in the above comments about `setGlobalEmbeddingColors`, which sets the global colors object. If `color` is not a function call it does not re-apply. See `SdkThemeProvider.ts`

But yes if we add `admin-navbar` to Mantine theme colors then we don't need the () => ...` hack

```ts
  const themeOverride = useMemo(() => {
    // !! Mutate the global colors object to apply the new colors.
    // This must be done before ThemeProvider calls getThemeOverrides.
    setGlobalEmbeddingColors(theme?.colors, appColors);

    return theme && getEmbeddingThemeOverride(theme);
  }, [appColors, theme]);
```

Reply:
Ok, that makes sense. I feel that eventually there should be only one way of dealing with things. By introducing that `setGlobalEmbeddingColors` function, we assume we'll keep using `color` along side `theme.fn.themeColor` which is a bit confusing to me.

Do you envision we keep using both `color` and `theme.fn.themeColor`? I feel like we should just use CSS custom properties e.g. --mb-color-brand and `theme.fn.themeColor` when we need to use util like `darken` and `lighten`.

What do you thinK?

Reply:
@WiNloSt Yep, I think we should advocate for CSS variables first when possible, then `theme.fn.themeColor` secondary. Let's try to not use `color(...)` when we can.

That `setGlobalEmbeddingColors` is intended to be a temporary solution to aid us in migration, not a permanent one.

Reply:
Agree! With the latest discussions on Mantine colors in mind, `() => color("admin-navbar")` is good.